src/components/oppfolgingsoppgave/OppfolgingsoppgaveModal.tsx

```diff
@@ -96,27 +96,48 @@ export const OppfolgingsoppgaveModal = ({
   toggleOpen,
   existingOppfolgingsoppgave,
 }: Props) => {
-  const [isGeneralFormError, setIsGeneralFormError] = useState(false);
+  const [isFormError, setIsFormError] = useState(false);
+  const [isFristTouched, setIsFristTouched] = useState(false);
   const createOppfolgingsoppgave = useCreateOppfolgingsoppgave();
   const editOppfolgingsoppgave = useEditOppfolgingsoppgave(
     existingOppfolgingsoppgave?.uuid
   );
   const {
     register,
-    formState: { errors },
+    formState: { errors, touchedFields },
     handleSubmit,
     setValue,
     watch,
-  } = useForm<FormValues>();
+  } = useForm<FormValues>({
+    defaultValues: {
+      beskrivelse: existingOppfolgingsoppgave?.tekst ?? "",
+      frist: existingOppfolgingsoppgave?.frist ?? null,
+    },
+  });
+  const watchedValues = watch();
   const isEditMode = !!existingOppfolgingsoppgave;
 
+  const isFormEdited = useCallback(
```

Comment:
Kan man bruke noe fra https://react-hook-form.com/docs/useformstate her? `isDirty`?

Reply:
`isDirty` sjekker bare om noe er endret. Ikke om de er like😐 Så man kan ende opp med akkurat de samme verdiene som default.

Reply:
Er det egentlig så viktig å håndtere dette i frontend (validering og ikke sende noe dersom det er uendret)? Hvis det blir veldig komplekst kan vi jo vurdere å bare la backend sjekke om ting er uendret returnere OK uten å lage ny versjon :thinking:

Reply:
Jeg tror en sånn type logikk blir vanskelig å forstå i fremtiden. Det blir vanskelig å koble den koden i backenden til et form som vi måtte skrive litt ekstra kode for å få til å fungere. Så jeg hadde heller gått for litt manuell logikk i frontend 😊